### PR TITLE
Altered 2 files

### PR DESCRIPTION
**Pull request by kaazima**
Following are the modified files:
• Positive or Negative.sh
• Smallest Number.sh

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
```console
$ ./Smallest\ Number.sh
Enter the total number of elements:
0
The smallest number among the 0 numbers is 10000
```

**Comment by aravindvnair99** [comment]
Also, `What is your name?` can be removed.

**Comment by kaazima** [comment]
Okay.

File: Source Codes/Smallest Number.sh [hunk 1]
```diff
@@ -4,14 +4,32 @@
...
+		while [[ ! "$num" =~ ^[+-]?[0-9]+$ ]];do
+			echo "Please enter only numbers"
+			read -r num
+		done
+		if [ "$num" -lt "$small" ]; then
+			small=$num
+		fi
+		i=$((i + 1))
+	done
+	echo "The smallest number among the $n numbers is $small"
```
Comment (aravindvnair99) [inline Source Codes/Smallest Number.sh]:
```suggestion
	echo "The smallest number is $small"
```

`The smallest number among the 1 numbers is 1` is wrong. Use this instead.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**